tools/cli/workflowCommands.go

```diff
@@ -1816,9 +1834,11 @@ func getLastDecisionCompletedID(ctx context.Context, domain, wid, rid string, fr
 			return "", 0, printErrorAndReturn("GetWorkflowExecutionHistory failed", err)
 		}
 		for _, e := range resp.GetHistory().GetEvents() {
-			if e.GetEventType() == shared.EventTypeDecisionTaskCompleted {
+			if e.GetEventType().Equals(decisionType) {
 				decisionFinishID = e.GetEventId()
+				return resetBaseRunID, decisionFinishID, nil
 			}
+			decisionFinishID = e.GetEventId()
```

Comment:
Should it be `e.GetEventId()+1`?  this decisionFinishID is exclusive in reset.

Reply:
updated